takwimu/static/js/charts.js

```diff
@@ -1092,6 +1108,10 @@ function Chart(options) {
 
     chart.showData = chart.actionLinks
         .append("a")
+        .attr("ga-on", "click")
+        .attr("ga-event-category", "Data")
+        .attr("ga-event-action","Show Data")
```

Comment:
Not a fan of action names with spaces: `ShowData`, `Show-Data`, etc. are better... Or if we can find a single word that roughly mean the same as `Show Data`

Reply:
Will do with `ShowData`.